### PR TITLE
style(vars): Add 2.0 color palette

### PR DESCRIPTION
**Pull request by glynnis**
https://jira.rax.io/browse/FRMW-249

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by bry4n** [comment]
LGTM

File: src/styles/vars.less [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+@gray-300: #757575;
+@gray-400: #9e9e9e;
+@gray-500: #bdbdbd;
+@gray-600: #d8d8d8;
+@gray-700: #e0e0e0;
+@gray-800: #e7e7e7;
+@gray-900: #eeeeee;
+@gray-950: #f5f5f5;
+@gray-975: #fafafa;
+@white: #ffffff;
```
Comment (CITguy) [inline src/styles/vars.less]: please remove the `@white` defined on line 53

**Comment by glynnis** [comment]
Got @ElementE 's eyes on this to check all my hex codes, checking the design box for him

File: src/styles/vars.less (hunk 1 above)
Comment (glynnis) [inline src/styles/vars.less]: Fixed!

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
[FRMW-259](https://jira.rax.io/browse/FRMW-259) created to capture documentation of these new variables.

**Pull request merged**